### PR TITLE
Export missing variables for trade and policy

### PR DESCRIPTION
**Pull request by JulianGeis**
Add missing variables:
for trade:

<google-sheets-html-origin><style type="text/css"></style>
2 | energy (supply) | Trade\|Primary Energy\|Biomass\|Volume | xx | not modelled | PJ/yr | net exports of solid, unprocessed biomass, at the global level these should add up to the trade losses only
-- | -- | -- | -- | -- | -- | --
2 | energy (supply) | Trade\|Secondary Energy\|Electricity\|Volume | xx | added  | PJ/yr | net exports of electricity, at the global level these should add up to the trade losses only
2 | energy (supply) | Trade\|Secondary Energy\|Hydrogen\|Volume | xx |  added | PJ/yr | net exports of hydrogen, at the global level these should add up to the trade losses only
2 | energy (supply) | Trade\|Secondary Energy\|Liquids\|Hydrogen\|Volume | xx | not modelled? | PJ/yr | net exports of liquid hydrocarbon fuels from renewable hydrogen and renewable CO2 (green synfuels), at the global level these should add up to the trade losses only
2 | energy (supply) | Trade\|Secondary Energy\|Gases\|Hydrogen\|Volume | xx | can not be distinguished? | PJ/yr | net exports of gaseous hydrocarbon fuels from renewable hydrogen and renewable CO2 (green syngases), at the global level these should add up to the trade losses only

<google-sheets-html-origin><style type="text/css"></style>
Trade\|Primary Energy\|Coal\|Volume | x | not modelled? | PJ/yr | net exports of coal, at the global level these should add up to the trade losses only
-- | -- | -- | -- | --
Trade\|Primary Energy\|Gas\|Volume | x | not modelled? | PJ/yr | net exports of natural gas, at the global level these should add up to the trade losses only
Trade\|Primary Energy\|Oil\|Volume | x | not modelled? | PJ/yr | net exports of crude oil, at the global level these should add up to the trade losses only

and policy:

<google-sheets-html-origin><style type="text/css"></style>
policy | Price\|Carbon\|ETS | x | same as global target? | EUR2020/t CO2 | Price of carbon within the EU ETS
-- | -- | -- | -- | -- | --
policy | Price\|Carbon\|EU-wide Regulation Non-ETS | x | not modelled? | EUR2020/t CO2 | Price of carbon implied by EU-wide Non-ETS sector target
policy | Price\|Carbon\|National Climate Target | x | same as national target? | EUR2020/t CO2 | Price of carbon due to national policies for all sectors - either from a direct carbon price, or as result of a national climate target
policy | Price\|Carbon\|National Climate Target Non-ETS | x | not modelled? | EUR2020/t CO2 | Price of carbon due to national policies for the non-ETS sectors - either from a direct carbon price, or as result of a national climate target for these sectors

Results:
image
image

### CODE REVIEW

**Comment by lindnemi** [comment]
Looking good so far. Two more points:

- [ ] `Trade|Secondary Energy|Liquids|Hydrogen|Volume` can be modeled with the `DE renewable oil -> EU oil` and vice versa links
- [ ] `Trade|Primary Energy|Gas|Volume` can be modeled with cross-border gas links. I would addtionally multiply with the EU-wide fossil gas share since the descirption mentions "natural gas"

**Review by lindnemi — COMMENTED** [review]
see comment above

**Review by toniseibold — APPROVED** [review]

File: workflow/scripts/export_ariadne_variables.py [hunk 1]
```diff
@@ -2971,6 +2972,74 @@ def get_policy(n):
 
     var["Price|Carbon"] = \
         -n.global_constraints.loc["CO2Limit", "mu"] - n.global_constraints.loc["co2_limit-DE", "mu"]
+    
+    var["Price|Carbon|ETS"] = \
+        -n.global_constraints.loc["CO2Limit", "mu"]
+
```
Comment (lindnemi) [inline workflow/scripts/export_ariadne_variables.py:2978]: This is not really ETS unfortunately

**Comment by lindnemi** [comment]
Codewise this PR looks good now. However, i am still a bit worried about the large swings in trade volume in later years:

After merging main:
trade

**Pull request merged**